qt/aqt/deckoptions.py

```diff
@@ -57,15 +60,58 @@ def _setup_ui(self) -> None:
 
     def _on_bridge_cmd(self, cmd: str) -> None:
         if cmd == "deckOptionsReady":
+            self.ready = True
             gui_hooks.deck_options_did_load(self)
 
+    def closeEvent(self, evt: QCloseEvent) -> None:
+        if self._close_event_has_cleaned_up:
+            evt.accept()
+            return
+        evt.ignore()
+        self.ifCanClose()
+
+    def _close(self):
+        """Close. Ensure the closeEvent is not ignored."""
+        self._close_event_has_cleaned_up = True
+        self.close()
+
+    def ifCanClose(self):
```

Comment:
We can't avoid camelCase for the Qt methods, but our own methods like this one and the ones embedded inside it, are better done with snake_case.